requirements.txt

```diff
@@ -1 +1,2 @@
 pipenv == 2023.11.15
+requests == 2.31.0
```

Comment:
we use Pipenv, so it should be added Pipfile and Pipfile.lock, instead of requirements.txt

Reply:
Fixed!